rs/bitcoin/ckbtc/minter/src/main.rs

```diff
@@ -291,33 +291,44 @@ fn http_request(req: HttpRequest) -> HttpResponse {
             None => 0,
         };
 
-        let mut entries: Log = Default::default();
-        for entry in export_logs(&ic_ckbtc_minter::logs::P0) {
-            entries.entries.push(LogEntry {
-                timestamp: entry.timestamp,
-                counter: entry.counter,
-                priority: Priority::P0,
-                file: entry.file.to_string(),
-                line: entry.line,
-                message: entry.message,
-            });
-        }
-        for entry in export_logs(&ic_ckbtc_minter::logs::P1) {
-            entries.entries.push(LogEntry {
-                timestamp: entry.timestamp,
-                counter: entry.counter,
-                priority: Priority::P1,
-                file: entry.file.to_string(),
-                line: entry.line,
-                message: entry.message,
-            });
+        let mut log: Log<Priority> = Default::default();
+
+        match req.raw_query_param("priority").map(Priority::from_str) {
+            Some(Ok(priority)) => match priority {
```

Comment:
nit: could this be simplified to
```suggestion
            Some(Ok(priority)) => log.push_logs(priority)
```

Reply:
Good point. Done.